src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -723,7 +760,7 @@ public override String ReadLine()
                         String s;
                         if (sb != null) {
                             sb.Append(charBuffer, charPos, i - charPos);
-                            s = sb.ToString();
+                            s = this.GetStringAndReleaseSharedStringBuilder(sb);
```

Comment:
Is there a plan to create a simple style guide in the wiki? It would help to avoid few iterations of review...